modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -1176,13 +1178,13 @@ private ExchangeType onClusterStateChangeRequest(boolean crd) {
                     if (log.isInfoEnabled()) {
                         log.info("Successfully activated caches [nodeId=" + cctx.localNodeId() +
                             ", client=" + kctx.clientNode() +
-                            ", topVer=" + initialVersion() + "]");
+                            ", topVer=" + initialVersion() + "] New state: " + req.state());
```

Comment:
Please keep the format for new field
[topVer=..., newState=ACTIVE]

Reply:
done